libs/saml2-web-sso/test/Test/SAML2/WebSSO/APISpec.hs

```diff
@@ -205,6 +205,8 @@ spec = describe "API" $ do
             (Proxy @APIAuthResp')
             (authresp' Nothing defSPIssuer defResponseURI (HandleVerdictRedirect (simpleOnSuccess SubjectFoldCase)))
 
+    -- TODO: test email, display name with / without unicode.  something else?
+
```

Comment:
```suggestion
```

i think this wouldn't increase test coverage significantly; unicode handling is (now) happening on a lower level.